vortexor/Readme.md

```diff
@@ -0,0 +1,157 @@
+The vortexor is a service which can be used to offload receiving transaction
+from the public, doing signature verifications and deduplications from the core
+validator which can focus on processing and executing the transactions. The
+filtered transactions can be forwarded to the validators linked with the
+vortexor. This vortexor makes the TPU transaction ingestion more scalable
+compared to single node solution.
+
+The archietecture diagram of the Vorexor with the relationship to the validator.
+
+                    +--------------------- VORTEXOR ------------------------+
+                    |                                                       |
+                    |   +-------------+        +--------------------+       |
+        TPU -->     |   | TPU Streamer| -----> | SigVerifier/Dedup  |       |
+        /QUIC       |   +-------------+        +--------------------+       |
+                    |        |                   |                          |
+                    |        v                   v                          |
+                    |  +----------------+     +------------------------+    |
+                    |  | Subscription   |<----| VerifiedPacketForwarder|    |
+                    |  | Management     |     +------------------------+    |
+                    |  +----------------+            |                      |
+                    +--------------------------------|----------------------+
+                                ^                    |
+    heart beat/subscriptions    |                    v
+                    +-------------------- AGAVE VALIDATOR ------------------+
+                    |                                                       |
+                    |  +----------------+      +-----------------------+    |
+Validator Config->  |  | Subscription   |      | VerifiedPacketReceiver|    |
```

Comment:
FYI - looks like this diagram formatting got messed up: https://github.com/lijunwangs/solana/tree/tpu_vortexor/vortexor

Reply:
Will fix it up

Reply:
Fixed. Looks at least need to start with 4 white spaces to format correctly